packages/codemods/README.md

```diff
@@ -0,0 +1,43 @@
+# codemods
+
+A collection of codemods for Helios components.
+
+## Usage
+
+To run a specific codemod from this project, you would run the following:
+
+```
+npx codemods <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
+
+# or
+
+yarn global add codemods
+codemods <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
+```
+
+## Local Usage
+```
+node ./bin/cli.js <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
+```
+
+## Transforms
+
+<!--TRANSFORMS_START-->
+* [v3/dropdown](transforms/v3/dropdown/README.md)
+<!--TRANSFORMS_END-->
```

Comment:
We should probably group them together (unless this is automated, which I suspect)
```suggestion
<!--TRANSFORMS_START-->
### v3.0.0
* [v3.0.0/dropdown](transforms/v3.0.0/dropdown/README.md)
<!--TRANSFORMS_END-->
```

Reply:
yes, it's unfortunately automated by running the `update-docs` command you touched on above.